tests/test_sqlite_duckdb_adapter.py

```diff
@@ -308,41 +340,49 @@ def test_prepare_insert_sql():
         (1000, 0, 1000),
     ],
 )
-def test_batch_size(tmp_path: Path, batch_size: int, expected_first: int, expected_second: int) -> None:
+@sqlite_duckdb_parametrize
+def test_batch_size(
+    tmp_path: Path,
+    batch_size: int,
+    expected_first: int,
+    expected_second: int,
+    db: Database,
+) -> None:
     """Test that batch_size is respected when writing records."""
     records = generate_records(batch_size + 100)
     db_path = tmp_path / "records.db"
-    with RecordWriter(f"sqlite://{db_path}?batch_size={batch_size}") as writer:
+    with RecordWriter(f"{db.scheme}://{db_path}?batch_size={batch_size}") as writer:
         # write a single record, should not be flushed yet if batch_size > 1
         writer.write(next(records))
 
         # test count of records in table (no flush yet if batch_size > 1)
-        with sqlite3.connect(db_path) as con:
-            x = con.execute("select count(*) from `test/record`")
+        with db.connector.connect(str(db_path)) as con:
+            x = con.execute('select count(*) from "test/record"')
```

Comment:
```suggestion
            x = con.execute('SELECT COUNT(*) FROM "test/record"')
```
Might be nice to be consistent.